language-configuration-erb.json

```diff
@@ -0,0 +1,31 @@
+{
+	"comments": {
+		"blockComment": [ "<%#", "%>" ]
+	},
+	"brackets": [
+		["{", "}"],
+		["[", "]"],
+		["(", ")"],
+		["<", ">"]
+	],
+	"autoClosingPairs": [
+		["{", "}"],
+		["[", "]"],
+		["(", ")"],
+		["`", "`"],
+		{ "open": "'", "close": "'", "notIn": ["string", "comment"] },
+		{ "open": "\"", "close": "\"", "notIn": ["string"] },
+		["<", ">"],
+		{ "open": "%", "close": "%", "notIn": ["string", "comment"] }
+	],
+	"surroundingPairs": [
+		["{", "}"],
+		["[", "]"],
+		["(", ")"],
+		["\"", "\""],
+		["'", "'"],
+		["`", "`"],
+		["<", ">"],
+		["%", "%"]
```

Comment:
It'll be `<%>`
It's either 

    ["<", ">"],
    { "open": "%", "close": "%", "notIn": ["string", "comment"] }

or

     ["<%", "%>"]


For HTML I think it's not going to be a problem since it's handled by emmet.